### ISSUE TITLE
Releasing new version

### ISSUE THREAD

**Issue by pomali** [viewflow/django-fsm#298]
I would love having new version with fixes for django 4.1+ (eg. #289).   It would greatly simplify my dependency management :)

Is there anything I can do to help with new release?

**Comment by pmyjavec** [comment]
Is this project dead? I'm guessing so?

**Comment by kmmbvnr** [comment]
It's replaced by Django-viewflow fsm module https://docs.viewflow.io/fsm/index.html

**Comment by pmyjavec** [comment]
@kmmbvnr ok so we should migrate away from this project altogether ? 

Any reason why this project still exists and is not archived, maybe not a bad idea to update the README.md?

**Comment by ds-clearago** [comment]
I'm also not clear as to what's going to happen to this library. Looking at the Readme of this and viewflow, it seemed like django-fsm is still being developed in parallel as the open source limited features variant of paid viewflow. But the above statement sounds very different.

Does that mean that PRs like https://github.com/viewflow/django-fsm/pull/297 are never going to be merged and released within this library and that starting a new project with django-fsm is not a good idea since it's effectively deprecated and instead people should be moving to viewflow?

Or are you @kmmbvnr going to focus your personal time and effort on viewflow (since this rewards your time with the license fee)? Is this repo then going to be archived or is this library potentially going to move to https://jazzband.co/ (where [django-fsm-log](https://github.com/jazzband/django-fsm-log) already is maintained)?

**Comment by jiaulislam** [comment]
Couldn't agree more with @ds-clearago . 

Are there any better alternative available for now ? I looked upon [transitions](https://github.com/pytransitions/transitions), but I don't know if I want to use that library either as there is no proper documentation. Also it does not fit in django ecosystem it says something about [django-transitions](https://github.com/PrimarySite/django-transitions). But again its not maintained anymore I guess. Anyone has better in mind ?

**Comment by ds-clearago** [comment]
@kmmbvnr Can you please elaborate on your plans with https://github.com/viewflow/django-fsm vs. https://docs.viewflow.io/fsm/index.html (https://github.com/viewflow/viewflow/tree/main/viewflow/fsm)

Looking at 
* https://github.com/viewflow/django-fsm/blob/master/CHANGELOG.rst
* https://github.com/viewflow/viewflow/blob/main/CHANGELOG.rst
* https://github.com/viewflow/django-fsm/blob/master/README.md
* https://github.com/viewflow/viewflow/blob/main/README.md
* https://github.com/viewflow/django-fsm/graphs/contributors
* https://github.com/viewflow/viewflow/graphs/contributors
* https://github.com/viewflow/django-fsm/tags
* https://github.com/viewflow/viewflow/tags
* https://github.com/viewflow/viewflow/pulls
* https://github.com/viewflow/viewflow/pull/386
* https://github.com/viewflow/viewflow/issues/344
* https://github.com/viewflow/django-fsm/pull/297

it's only clear that https://github.com/viewflow/viewflow has more recent activity (last tag in django-fsm 08.2022 vs viewflow 09.2023) and is getting more attention. But it's not certainly visible, whether either library is actively maintained at the moment (even viewflow has simple PRs open from July).

As I've said, I'm in principle willing to become a Viewflow pro customer, but then it has to be clear what the situation is. Can you please comment something on this, as you're the main contributor to django-fsm and the only contributor to viewflow?

Thank you very much.
Best,
Dennis

**Comment by Natim** [comment]
What do you think about moving its maintenance to [Jazzband](https://jazzband.co/about) so that we can keep maintaining it?

**Comment by RemiDesgrange** [comment]
@kmmbvnr the workflow and code logic in viewflow is quite different from django-fsm. Same as @ds-clearago , can you elaborate on your plan for django-fsm.

Thx.

**Comment by ds-clearago** [comment]
@Natim The Jazzband issue is being discussed already in https://github.com/viewflow/django-fsm/issues/285 and https://github.com/viewflow/django-fsm/issues/298
Furthermore, looking at stories like this https://github.com/django-guardian/django-guardian/issues/759, it doesn't seem to be as simple as "if no maintainer's here, then there will be plenty there" especially since the transition also requires effort + at least one project lead willing to be named as such on Jazzband's side.

So I think it comes down to what @kmmbvnr is planning with django-fsm and viewflow.

Looking at https://github.com/kmmbvnr/django-material being archived on Apr 12, 2023 in favor of viewflow but also no answer in https://github.com/viewflow/viewflow/issues/400 since Oct 12th my best guess is that he's busy at the moment with other things in life rather than these packages...

**Comment by Natim** [comment]
> at least one project lead willing to be named as such on Jazzband's side.

We are ok to do the job with @pfouque

**Comment by ds-clearago** [comment]
While there is no reply here, there has been a relevant comment on this elsewhere:
In https://github.com/viewflow/viewflow/wiki/Django%E2%80%90Material-and-Viewflow-1.x-migration-Guide#conceptual-changes it says
> Django-fsm and django-material have been merged into django-viewflow. Now, you only need to install django-viewflow (or django-viewflow-pro).

**Comment by RemiDesgrange** [comment]
Thanks @ds-clearago 

django-viewflow v2 is still in beta and published as a pre-release on Pypi. Lastest release is from september 2023. 

Even if I like the decoupling between the "flow" (state machine) and actual models in viewflow vs having heavy models in django-fsm, both options seem valid to me. Can't see why we couldn't have django-fsm in parrallel of django-viewflow. I understand perfectly if @kmmbvnr doesn't want to maintain this library and viewflow. 
Amazing job has been done. Maybe we could continue it? Here or elsewhere.

**Comment by kmmbvnr** [comment]
Guys, Viewflow would be released soon as a stable.

viewflow.fsm is the django-fsm (3.0) 

django-fsm 2.x was released as 2014 has several design flaws and quite obsolete
